### PR TITLE
docs: mention new extension `vike-react-antd`

### PR DESCRIPTION
**Pull request by phonzammi**
Docs updated :

> * vike.dev/antd
> * vike.dev/extensions

Please take a look and review it. 

I'm not sure about : 
> * vike.dev/releases/2024-09 (I should rename and publish it at some point 😅)

### CODE REVIEW

**Comment by brillout** [comment]
Add it here: https://vike.dev/releases/2024-09#new-bati-integrations

File: docs/pages/antd/+Page.mdx [hunk 1]
```diff
@@ -3,9 +3,22 @@ import { Link } from '@brillout/docpress'
...
+
 Examples of using Vike with [Ant Design](https://ant.design):
+ - <Example timestamp="2024.11" repo="phonzammi/vike-react-antd-example" /> - Using <Link href="/vike-react">`vike-react`</Link>
  - <Example timestamp="2024.08" repo="lewis-fidlers/vike-react-antd" /> - Using <Link href="/vike-react">`vike-react`</Link>
 
 ## See also
 
-- [vike#1804 Some help with injecting styles](https://github.com/vikejs/vike/discussions/1804)
+- [Ant Design > Server Side Rendering](https://ant.design/docs/react/server-side-rendering)
+- [vike#1804 Some help with injecting styles](https://github.com/vikejs/vike/discussions/1804)
```
Comment (brillout) [inline docs/pages/antd/+Page.mdx:24]: Wonderful idea :dizzy:

File: docs/pages/antd/+Page.mdx [hunk 2]
```diff
@@ -3,9 +3,22 @@ import { Link } from '@brillout/docpress'
 
 <CommunityNote />
 
+To use [Ant Design](https://ant.design/), you can use [`vike-react-antd`](https://github.com/vikejs/vike-react/tree/main/packages/vike-react-antd) for an automatic integration with Vike.
+
+> The `vike-react-antd` extension requires <Link href="/vike-react">`vike-react`</Link>.
+
+## Manual integration
+
+If you use <Link href="/vike-react">`vike-react`</Link>, you can manually integrate Ant Design using combination of these :
```
Comment (brillout) [inline docs/pages/antd/+Page.mdx:12]:
```suggestion
If you use <Link href="/vike-react">`vike-react`</Link>, you can manually integrate Ant Design using combination of these:
```
Intervention++

Comment (phonzammi) [reply docs/pages/antd/+Page.mdx]: Thanks 😅

**Comment by phonzammi** [comment]
sorry for the typo 😅 

LGTM ?

**Comment by brillout** [comment]
LGTM! Please **squash** and merge (see drop down options).

**Pull request merged**